internal/app/networkd/pkg/nic/nic.go

```diff
@@ -312,18 +312,26 @@ func (n *NetworkInterface) Renew() {
 // address TTL. If that fails, we'll continue to attempt to retry every
 // halflife.
 func (n *NetworkInterface) renew(method address.Addressing) {
+	const minRenewDuration = 5 * time.Second // protect from renewing too often
```

Comment:
this prevents code from entering tight loop on sending endless stream of DHCP requests